easybuild/easyconfigs/f/FigureGen/FigureGen-51-foss-2019a.eb

```
@@ -0,0 +1,39 @@
+easyblock = 'CmdCp'
+
+name = 'FigureGen'
+version = '51'
+
+homepage = 'https://ccht.ccee.ncsu.edu/figuregen-v-49/'
+description = """  FigureGen is a Fortran program that creates images for ADCIRC files.
+It reads mesh files (fort.14, etc.), nodal attributes files (fort.13, etc.) and output 
+files (fort.63, fort.64, maxele.63, etc.). It plots contours, contour lines, and vectors. 
+Using FigureGen, you can go directly from the ADCIRC input and output files to a 
+presentation-quality figure, for one or multiple time snaps.  """
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = [{'download_filename': 'master.zip', 'filename': SOURCE_ZIP}]
+checksums = ['4199816638cf95ccd661eb0509c46d80c9dae986b2053a2db23cce293dcfa328']
+
+github_account = 'ccht-ncsu'
+files_to_copy = [
+    (['*'], '.'),
+]
+
+cmds_map = [('.*', 'mpif90 FigureGen.F90 -DCMPI -o FigureGen.exe')]
```

With EasyBuild the compiler and compiler flags should not be hardcoded, as those are determined by the toolchain defined in the easyconfig (`foss` in this case). If you want to use `mpif90`, you can do so by setting the following variables in your compilation command
```suggestion
cmds_map = [('.*', '$MPIF90 $F90FLAGS FigureGen.F90 -DCMPI -o FigureGen.exe')]
```
Keep in mind that for these variables to work, you will need to also add the `usempi` option in the toolchain (see above)